src/ja/server/proxy/command_handler.py

```diff
@@ -60,9 +64,14 @@ def _user_is_admin(self, user: str) -> bool:
         return self._admin_group in groups
 
     def _execute_command(self, command: ServerCommand) -> Dict[str, object]:
-        return command.execute(self._database).to_dict()
+        logger.info("executing server command")
+        logger.debug("%s" % str(command))
+        r_dict = command.execute(self._database)
+        logger.debug("%s" % str(r_dict))
+        return r_dict.to_dict()
 
     def _process_worker_message(self, command_dict: Dict[str, object], type_name: str, user: str) -> Dict[str, object]:
+        logger.info("processing worker message")
```

Comment:
if you look at the logic below, worker_message is called always, and fails if the param is not a user message ...
So I do not think it makes sense to print something here, as well as below.